tests/unit/test_resolve_local_ids.py

```diff
@@ -0,0 +1,18 @@
+import json
+import unittest
+
+from dm_cli.utils.resolve_local_ids import resolve_local_ids_in_document
+
+
+class ResolveLocalIdsTest(unittest.TestCase):
+    def test_resolve_local_ids_in_document(self):
+        file = open("tests/unit/resolve_local_ids_test_data/carRentalCompany.json")
```

Comment:
Tip use: with open("hello.txt") as my_file:
    .....

Then the close method is not need to be called explicit.

Reply:
Ah yes that works. Thought initially that `with` created a scope.